vcd/resource_vcd_org_vdc.go

```diff
@@ -208,6 +208,13 @@ func resourceVcdOrgVdc() *schema.Resource {
 				Required:    true,
 				Description: "When destroying use delete_recursive=True to remove the vdc and any objects it contains that are in a state that normally allows removal.",
 			},
+			"metadata": {
+				Type:        schema.TypeMap,
+				Optional:    true,
+				Description: "key and value for Org VDC metadata ",
```

Comment:
What about "Key and value pairs for Org VDC metadata" (capital letter, plural reference and no space at the end)?

Reply:
changed